src/components/AnimationComponent.ts

```diff
@@ -1,3 +1,44 @@
+export interface AnimationSprite {
```

Comment:
Jag funderar på om vi kan göra det här ännu bättre? Iom att alla animatinos alltid är på formen: gameObject-action-direction borde vi kunna slippa skicka in hela strängen player-walk-down å istället skapa ett AnimationsObjekt som tar in antalet sprites i varje kategori eller nått sånt